### DISCUSSION TITLE
Added: LDM Notes for April 5th, 2021

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#4625] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2021/LDM-2021-04-05.md

Agenda:

1. Interpolated string improvements
2. Abstract statics in interfaces

**Comment by Unknown6656** [comment]
@333fred Do you know by any chance whether discriminated unions will be ready for C#10?
The last couple of LDM notes did not really seem to mention the topic and I was wondering what the current state of progress on discr. unions is.....

**Reply by 333fred** [reply]
I can be reasonably certain they will not be ready.

**Reply by orthoxerox** [reply]
That's not unexpected, but still sad to hear.

**Reply by Unknown6656** [reply]
Oh no :(((
I've been looking forward to discriminated unions ever since the initial proposal in C#6.

**Reply by Unknown6656** [reply]
I've got Mads "official" confirmation that discriminated unions will not make it into C#10: https://twitter.com/MadsTorgersen/status/1387079125676875776

**Reply by spydacarnage** [reply]
Understandable, but still "boo!" :P

**Reply by Unknown6656** [reply]
@spydacarnage agreed :P

**Reply by WashingtonARamos** [reply]
> 
> 
> I've got Mads "official" confirmation that discriminated unions will not make it into C#10: https://twitter.com/MadsTorgersen/status/1387079125676875776

So sad. I really hope they can work it out for DUs to make it into C# 11, then. If there's *one* thing I miss from my short trip to functional languages is DUs.
